### ISSUE TITLE
Add customization of the data pushed by the F5 equipment.

### ISSUE THREAD

**Issue by legea** [F5Networks/f5-telemetry-streaming#5]
Please add the customization of the data pushed.

Especially I need to gather statistics with the tmctl command like these ones for instance,

```
tmctl -d blade -i tmm/ndal_rx_stats
tmctl -d blade -i tmm/ndal_tx_stats
```	
It would by great to be able of selecting the tmctl tables from the statistics subsystem that we would need to export by the telemetry service.

**Comment by jsevedge** [comment]
@legea Is the list of stats needed here always changing?  How many different stats are you currently gathering via tmctl?

**Comment by legea** [comment]
Indeed the list is always changing because it is use case dependent.
I'm interested in the tmm/ndal table since few months, and soon it will probably be another.

For the moment I collect around 500 metrics via tmctl.

**Comment by focrensh** [comment]
Tracked with AT-290

**Comment by bnenjk84** [comment]
@jsevedge @focrensh - Would be great to be able to define/filter a list of stats and then select the output - IE send all stats to AWS, send only virtual server stats to elastic

**Comment by focrensh** [comment]
@bnenjk84 Please open a separate issue for this RFE and we will get it tracked. Thanks

**Comment by aknot242** [comment]
> @bnenjk84 Please open a separate issue for this RFE and we will get it tracked. Thanks

@focrensh or @bnenjk84 - was this RFE ever opened? I think it is a great idea, and if not I will post the RFE.

**Comment by dstokesf5** [comment]
It appears AUTOTOOL-290 was resolved as part of the 1.5.0 release. Closing this issue. If you are still experiencing an issue, please reopen this issue or create a new one.
